docker-entrypoint.sh

```diff
@@ -29,35 +29,35 @@ create_environment_file() {
     # App settings
```

Comment:
May I suggest to use a herefile to make it more easily copy-pastable?

```bash
cat <<EOF >> /var/www/html/.env
# App settings
APP_NAME="HortusFox"
APP_DEBUG=$APP_DEBUG
APP_BASEDIR="$APP_BASE_DIR"
APP_LANG="$APP_LANG"
APP_WORKSPACE="$APP_WORKSPACE"
APP_ENABLESCROLLER="$APP_ENABLE_SCROLLER"
APP_OVERLAYALPHA="$APP_OVERLAY_ALPHA"
APP_ENABLECHAT="$APP_ENABLE_CHAT"
APP_ONLINEMINUTELIMIT="$APP_ONLINE_MINUTE_LIMIT"
APP_SHOWCHATONLINEUSERS="$APP_SHOW_CHAT_ONLINE_USERS"
APP_SHOWCHATTYPINGINDICATOR="$APP_SHOW_CHAT_TYPING_INDICATOR"
APP_OVERDUETASK_HOURS=$APP_OVERDUE_TASK_HOURS
APP_CRONPW="$APP_CRON_PW"
APP_CRONJOB_MAILLIMIT=$APP_CRON_MAIL_LIMIT
APP_GITHUB_URL="https://github.com/danielbrendel/hortusfox-web"
APP_SERVICE_URL="https://www.hortusfox.com"
APP_ENABLEHISTORY=$APP_ENABLE_HISTORY
APP_HISTORY_NAME="$APP_HISTORY_NAME"

# Session
SESSION_ENABLE=true
SESSION_DURATION=31536000
SESSION_NAME=null

# Photo resize factors
PHOTO_RESIZE_FACTOR_DEFAULT=$PHOTO_RESIZE_FACTOR_DEFAULT
PHOTO_RESIZE_FACTOR_1=$PHOTO_RESIZE_FACTOR_1
PHOTO_RESIZE_FACTOR_2=$PHOTO_RESIZE_FACTOR_2
PHOTO_RESIZE_FACTOR_3=$PHOTO_RESIZE_FACTOR_3
PHOTO_RESIZE_FACTOR_4=$PHOTO_RESIZE_FACTOR_4
PHOTO_RESIZE_FACTOR_5=$PHOTO_RESIZE_FACTOR_5

# Database settings
DB_ENABLE=true
SMTP_PORT=$SMTP_PORT
SMTP_USERNAME="$SMTP_USERNAME"
SMTP_PASSWORD="$SMTP_PASSWORD"
SMTP_ENCRYPTION="$SMTP_ENCRYPTION"

# Logging
LOG_ENABLE=$LOG_ENABLE
EOF

```

Reply:
Looks more clean and better maintainable.

Reply:
I agree, I updated the entrypoint script and took a stab at updated the variable documentation in readme.md